api/src/test/java/org/openmrs/notification/db/hibernate/HibernateAlertDAOTest.java

```diff
@@ -32,11 +48,49 @@ public void runBeforeEachTest() {
 	 * @throws Exception
 	 */
 	@Test
-	public void shouldGetAlerts() {
-		
-		AlertService as = Context.getAlertService();
-		//System.out.println(as.getAllAlerts());
+	public void getAlert_shouldReturnSavedAlert() throws Exception {
+		Alert alert = createAlert();
+		Context.getAlertService().saveAlert(alert);
+		Alert savedAlert = Context.getAlertService().getAlert(1234);
+		Assert.assertEquals(savedAlert.getText(), "Testing the hibernate DAO");
 		
 	}
 	
+	@Test
+	public void saveAlert_shouldReturnNotNull() {
+
+		Alert alert = createAlert();
+		Context.getAlertService().saveAlert(alert);
+		Alert savedAlert = Context.getAlertService().getAlert(1234);
+		Assert.assertNotNull(savedAlert.getAlertId());
+
+	}
+	public void deleteAlert_shouldReturnNullAfterDeleting() {
```

Comment:
Same as here  line 68 the travis is complaining, so make it a test by adding annotation @Test.

Reply:
@sherrif10   i  must be dosing off lol,fixed

Reply:
No dosing here man